src/data/fin.lean

```diff
@@ -84,6 +84,7 @@ lemma eq_iff_veq (a b : fin n) : a = b ↔ a.1 = b.1 :=
 ⟨fin.mk.inj, λ h, by subst h⟩
 
 instance fin_to_nat (n : ℕ) : has_coe (fin n) nat := ⟨fin.val⟩
+instance fin_to_nat' (n : ℕ) : has_coe_t (fin n) nat := ⟨fin.val⟩
```

Comment:
Why is this necessary when `instance coe_base {a : Sort u} {b : Sort v} [has_coe a b] : has_coe_t a b` exists in core? Same question for `int.has_coe_t`, `instance : has_coe_t ℕ enat`, etc. Is this a priority issue? Should we raise the priority of `coe_base`?

Reply:
I didn't think about `coe_base` too much, I lowered the priority together with `coe_trans`.  But you're probably right, I'll try increasing the priority of `coe_base` to 950.  Then manual `has_coe_t` instances still take precedence, but we wouldn't need `fin_to_nat'`.